examples/LAMMPS_OPENFOAM/cpl/COUPLER.in

```diff
@@ -0,0 +1,131 @@
+###############################################################################
+#                                                                             #
+#           8888888 888b    888 8888888b.  888     888 88888888888            #
+#             888   8888b   888 888   Y88b 888     888     888                #
+#             888   88888b  888 888    888 888     888     888                #
+#             888   888Y88b 888 888   d88P 888     888     888                #
+#             888   888 Y88b888 8888888P"  888     888     888                #
+#             888   888  Y88888 888        888     888     888                #
+#             888   888   Y8888 888        Y88b. .d88P     888                #
+#           8888888 888    Y888 888         "Y88888P"      888                #
+#                                                                             #
+###############################################################################
+
+###############################################################################
+#
+#   density_cfd (float)
+#     - specifies the density for both continuum fluid
+#
+# -----------------------------------------------------------------------------
+DENSITY_CFD
+0.5
+
+FULL_OVERLAP
+.true.
+
+
+###############################################################################
+#
+#   overlap_extents (6 * integers)
+#     - define the global cell boundaries of the overlap region
+#
+#     1. icmin_olap
+#     2. icmax_olap
+#     3. jcmin_olap
+#     4. jcmax_olap
+#     5. kcmin_olap
+#     6. kcmax_olap
+#
+# -----------------------------------------------------------------------------
+OVERLAP_EXTENTS
+1
+8
+1
+4
+1
+8
+
+##############################################################################
+##
+##   Specify constraint information
+##
+##      1. constraint_algo 0-off, 1-OCT, 2-NCER, 3-Flekkoy 4-CV
+##      2. constraint_CVflag -- Use CV form of constraint 0-Off 1-On
+##      3. icmin_cnst
+##      4. icmax_cnst
+##      5. jcmin_cnst
+##      6. jcmax_cnst
+##      7. kcmin_cnst
+##      8. kcmax_cnst
+##
+## -----------------------------------------------------------------------------
+CONSTRAINT_INFO
+4
```

Comment:
I don't quiet understand what these parameters do, but the 8s are necessary; they were 32 before which wasn't compatible with the OpenFOAM case we have apparently

Reply:
Sorry, I missed these (my notification email was set to my old Imperial account). The constraint_info specifies which cells to apply the constraint on. The idea of CPL library is to use a single grid (the CFD one) for both MD and CFD regions and then apply constraints to a subset of the cells which overlap the MD domain. In this case, 32 was the number of cells in the CFD domain

Reply:
I completely forgot to mail you about it; apologies for that!
Looks like  something is wrong with my porting but didn't have enough time to properly debug the situation; will push for this over next week!